src/Pages/SignUp/SignUp.scss

```
@@ -0,0 +1,217 @@
+.SignUp {
+  height: 1104.62px;
+  margin-top: 130px;
+
+  .signUpHeader {
+    background-image: url("https://image.converse.co.kr/cmsstatic/bg-register-top-pc.jpg");
+    height: 260px;
+
+    .textBox {
+      display: flex;
+      flex-direction: column;
+      width: 830px;
+      height: 260px;
+      margin-left: 800px;
+      justify-content: center;
+      align-items: center;
+      text-align: center;
+      color: white;
```

```suggestion
    .textBox {
      width: 830px;
      height: 260px;
      display: flex;
      flex-direction: column;
      justify-content: center;
      align-items: center;
      margin-left: 800px;
      text-align: center;
      color: white;
```
1. 중요한 속성 먼저 배치해주시고, 유사한 속성끼리 묶어주세요
2. div 태그는 기본적으로 block 요소 입니다. 세로 방향으로 배열되는 것이 디폴트이므로 flex direction column은 대부분의 경우 필요 없습니다.